.github/workflows/ccpp.yml

```diff
@@ -20,8 +20,15 @@ jobs:
     env:
       build_dir: ./src/build
       dist_dir: ./src/local_third_party
+      build_deps: OFF
```

Comment:
Will this be turned back on automatically when creating an actual release?

Reply:
Yes, check "Check for full rebuild" step